subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -840,10 +871,18 @@ static bool send_request(enum http_method method, enum hawkbit_http_request type
 		break;
 
 	case HAWKBIT_CONFIG_DEVICE:
-		memset(&cfg, 0, sizeof(cfg));
+		struct hawkbit_cfg cfg = {0};
```

Comment:
What do you think about sending only the minimal info in default hawkBit implementation (i.e. the existing stuff like `VIN` (I forgot if `hwRevision` is strictly required)), and `merge` the custom attributes in an additional steps?

Reply:
`VIN` and `hwRevision` are both not strictly required by hawkBit. But I would still like to send `VIN` as our default configuration. `hwRevision` will be removed from the default configuration.